python/tvm/relay/op/nn/_nn.py

```diff
@@ -163,10 +163,17 @@ def compute_conv2d(attrs, inputs, out_type, target):
 
     def _get_out_depth():
         weight_shape = get_const_tuple(inputs[1].shape)
+        # NHWC layout
         if kernel_layout.startswith("HW"):
             return weight_shape[2] * weight_shape[3]
-        return weight_shape[0] * weight_shape[1]
-
+        # NCHW layout.
```

Comment:
Can we add an assertion for the layout format

Reply:
@tmoreau89 I think we shouldn't need do this check. Because this is a utility function and layout check is done before calling it. https://github.com/apache/incubator-tvm/blob/d900cd67f046640e39f2b4fcd9d9a7a18d7f7f3a/python/tvm/relay/op/nn/_nn.py#L181-L184

Reply:
Indeed you are correct; thanks for correcting